Lib/traceback.py

```diff
@@ -330,7 +330,7 @@ def walk_stack(f):
     current stack is used. Usually used with StackSummary.extract.
     """
     if f is None:
-        f = sys._getframe().f_back.f_back.f_back.f_back
+        f = sys._getframe().f_back.f_back
```

Comment:
```suggestion
        f = sys._getframe().f_back
```

Reply:
I think Python 3.10 was wrong too. This should just f_back.
See `print_stack()` and `format_stack()`.